src/main/java/com/epam/test_generator/services/StepService.java

```diff
@@ -159,33 +161,4 @@ public void removeStep(Long projectId, Long suitId, Long caseId, Long stepId) {
         caseVersionDAO.save(caze);
         suitVersionDAO.save(suit);
     }
-
```

Comment:
Let's mention this move in the PR message to not lose track of what's going on with cascade updates

Reply:
Done